library/core/src/ptr/const_ptr.rs

```diff
@@ -465,8 +465,9 @@ impl<T: ?Sized> *const T {
     /// If any of the following conditions are violated, the result is Undefined
     /// Behavior:
     ///
-    /// * Both the starting and resulting pointer must be either in bounds or one
-    ///   byte past the end of the same [allocated object].
+    /// * If the computed offset is non-zero, then both the starting and resulting pointer must be
+    ///   either in bounds or one byte past the end of the same [allocated object].
+    ///   (If it is zero, then the function is allays well-defined.)
```

Comment:
```suggestion
    ///   (If it is zero, then the function is always well-defined.)
```

ditto below

Reply:
Thanks!
This is actually from https://github.com/rust-lang/rust/pull/117329, I'll fix it there.